src/main/java/io/vertx/core/http/impl/MimeMapping.java

```diff
@@ -1014,7 +1014,10 @@ public class MimeMapping {
   }
 
   public static String getMimeTypeForExtension(String ext) {
-    return m.get(ext);
+    if (ext == null) {
+      return null;
+    }
+    return m.get(ext.toLowerCase());
```

Comment:
use HttpUtils#toLowerCase here instead, it is more performant

Reply:
get it